net/nebula/Makefile

```diff
@@ -69,53 +70,45 @@ define Package/nebula-service
   PKGARCH:=all
 endef
 
```

Comment:
again the URL should point to projects URL, not your downstream URL project. 

Anything else is unrelevant, I checked the URL.

1. How to install packages
Oh well, here we go again. Thats why there is LuCI, if you are familiar with SSH, then you know how to install packages. If not, go look at documentation (OpenWrt wiki)

2. Default settings, hm... shouldnt it be default?
Kinda obvious.

I dont see benefits or advantages to do the documentation, which are you doing. Sorry.

Reply:
You're entitled to your opinion, however an attentive reader would have noticed that how to install and default settings are not numbers 1 and 2, but numbers 3 and 4.

Reply:
Oh boy, you should really stop reinvent a wheel again.

Look here:
https://openwrt.org/docs/guide-developer/packages

What is in PKG_URL? Oh... yeah! Upstream project URL.

Reply:
And yet, the PKG_URL is not even present in the example Makefile.

I see no benefit for OpenWrt users to have the link pointing to a page which has very little relevance to the OpenWrt package.

I appreciate a lot of your other feedback on this and other packages, but your holy war against my documentation repository (of which you choose to ignore important sections) is bordering on harassment.